Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerExchangeInformation.ps1

```diff
@@ -92,7 +92,7 @@ function Invoke-AnalyzerExchangeInformation {
         }
 
         $displayValue = "$($exchangeInformation.BuildInformation.VersionInformation.ExtendedSupportDate.ToString("MMM dd, yyyy",
-            [System.Globalization.CultureInfo]::CreateSpecificCulture("en-US"))) - Please note of the End Of Life date and plan to migrate soon."
+            [System.Globalization.CultureInfo]::CreateSpecificCulture("en-US"))) - Please note the End Of Life date. Reference our blog for more information: https://aka.ms/HC-UpgradeToSE"
```

Comment:
Let's make it so only Exchange 2013/2016/2019 will throw this line, then everything else will default to what we previously had. Just in case we do release another version after SE.